modules/renderer/background_source.js

```diff
@@ -92,7 +93,9 @@ export function rendererBackgroundSource(data) {
 
     source.area = function() {
         if (!data.polygon) return Number.MAX_VALUE;  // worldwide
-        var area = d3_geoArea({ type: 'MultiPolygon', coordinates: [ data.polygon ] });
+        var area = data.polygon.reduce((acc, outerRing) => {
+            return acc + Math.abs(polygonArea({ type: 'Polygon', coordinates: [outerRing] }));
+        }, 0);
```

Comment:
> I'd double check whether ELI's GeoJSON handles polygons with holes properly. For years it didn't, so we have code in iD to workaround this issue. (I submitted a PR to ELI years ago to fix this and Stereo didn't want it.)

If I remember correctly, the oddest thing about ELI’s representation is that a polygon can have multiple outer rings. This change does seem to address the same case without affecting inner ring handling, but there should be a test about it if there isn’t already.